eng/Version.Details.xml

```diff
@@ -164,6 +164,10 @@
       <Uri>https://github.com/dotnet/runtime-assets</Uri>
       <Sha>758d8a04223a3238863dd28bce0ec2e20b3bb960</Sha>
     </Dependency>
+    <Dependency Name="System.Data.Common.TestData" Version="8.0.0-beta.23110.1">
```

Comment:
https://dnceng.visualstudio.com/public/_artifacts/feed/dotnet8-transport/NuGet/System.Data.Common.TestData/overview/8.0.0-beta.23113.1

According to dotnet8-transport, the latest number of this package is `8.0.0-beta.23113.1`.

Reply:
Not a problem from the chat discussion.